scripts/inserter.lua

```diff
@@ -82,6 +83,28 @@ function inserter.init()
   global.temp_inserter_settings = {}
 end
 
+function inserter.find_droplanes()
+  -- Find all compatible inserters
+  local has_droplane_gui = {}
+  for _, entity in pairs(game.get_filtered_entity_prototypes({ { filter = "type", type = "inserter" } })) do
+    if entity.name:sub(-#"-miniloader-inserter") == "-miniloader-inserter" then
```

Comment:
This could be simplified using `string.find`

Reply:
Simple as that is it is a little out of my factorio/lua comfort zone, i need examples to mimic, but in this case its probably just easier if you do it without me guessing several times what it should be, `Allow edits by maintainers` is on 😅

Reply:
I went ahead and made the change.